src/ngAnimate/animate.js

```diff
@@ -1194,18 +1194,22 @@ angular.module('ngAnimate', ['ng'])
           }
 
           return cache.promise = runAnimationPostDigest(function(done) {
-            var parentElement = element.parent();
-            var elementNode = extractElementNode(element);
-            var parentNode = elementNode.parentNode;
+            var cache, parentNode, parentElement, elementNode = extractElementNode(element);
+            var DOM_FRAGMENT = 11;
```

Comment:
This get removed when it was merged to master: https://github.com/angular/angular.js/commit/2c03a3574336ed814d020cf7ba36cee5b87e65b5